Hide side nav sub-links that aren't related to the 'active' page

yjukaku opened the pull request:
- Separates nav 'sub-links' into `.link-groups`
- Updates TOC generator to add `.active` class to the link-group that is `.active`, which effectively hides the non-active `.link-groups`

Unfortunately, I can't get this repo running locally, so I can't really test it. 😢  I just wanted to get it out there in case anyone who can test it out can do so.

Here is what I *imagine* it to look like:

![screen shot 2017-02-14 at 4 46 32 pm](https://cloud.githubusercontent.com/assets/908365/22950896/31d14c96-f2d5-11e6-8675-32b24ae028f2.png)


Closes #75

yjukaku committed:
hide side nav sub-links that aren't related to the 'active' page

apotonick commented:
Awesome! :heart_eyes: 

I'm starting to wonder, doesn't Foundation 6 come with collapsable menus, though?

yjukaku commented:
Ah, yeah, you're right.  But it looks like we'd have to change the UX a little bit - the header links in a Foundation accordion menu are used to just open the menu, but here we use header links to point to `/index` of the 'group'.  We could change the headers to just be accordion 'openers', and add an 'Home' link under each group.... What do you think?

apotonick commented:
Or we could add a button or link in the expanding menu, e.g. *Overview*. The top-level node links are very hard to see anyway and barely get clicked.

apotonick commented:
![image](https://cloud.githubusercontent.com/assets/10406/23013266/2d39c41e-f42a-11e6-8090-2ea60c75c951.png)

fran-worley commented:
For what it's worth, I like the idea of an overview page...

Sent from my iPhone

On 16 Feb 2017, at 08:26, Nick Sutterer <notifications@github.com<mailto:notifications@github.com>> wrote:


Or we could add a button or link in the expanding menu, e.g. Overview. The top-level node links are very hard to see anyway and barely get clicked.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/trailblazer/trailblazer.github.io/pull/83#issuecomment-280265001>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACNaJYalxafToo7Ml5v3-duVTbsKqYqJks5rdAg_gaJpZM4MBCUX>.

apotonick commented:
The overview pages are already there, e.g. `/gems/reform/index.html` :joy:

fran-worley commented:
I said for what it's worth... not a lot clearly!! 👍🤓

apotonick commented:
I no english :nerd_face:

yjukaku commented:
OK, so I moved the links around, added the necessary classes  to use the Foundation accordion menu (http://foundation.zurb.com/sites/docs/accordion-menu.html), and added 'Overview' links under each heading.

 Let me know how that works 👍 

EDIT

I also turned off the SCSS `$accordionmenu-arrows` variable - I didn't see it used anywhere else in the site, but I could definitely be wrong.

apotonick commented:
Thanks for your work @yjukaku - we now have a full-time frontend crack who implemented the accordion. Thanks :heart:
